src/java.desktop/share/classes/com/sun/java/swing/plaf/gtk/GTKLookAndFeel.java

```diff
@@ -865,6 +865,7 @@ public Object createValue(UIDefaults table) {
                           "ctrl ENTER", "press",
                  "ctrl released ENTER", "release"
             },
+            "RootPane.altPress", true,
```

Comment:
`RootPane.hideMnemonics` or `RootPane.showMnemonicsOnAltPress` could be a more descriptive property name.

Reply:
Removed this property now.